GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -793,22 +772,29 @@ public void ForceRefreshRevisions()
 
                 _gridView.SuspendLayout();
                 _gridView.SelectionChanged -= OnGridViewSelectionChanged;
+                _gridView.ClearSelection();
+                _gridView.Clear();
                 _gridView.Enabled = true;
                 _gridView.Focus();
                 _gridView.SelectionChanged += OnGridViewSelectionChanged;
                 _gridView.ResumeLayout();
 
                 if (_initialLoad)
                 {
-                    _initialLoad = false;
```

Comment:
What is the purpose of this? To restart the timer at parallel calls of ForceRefreshRevisions()?